.circleci/config.yml

```diff
@@ -22,10 +22,10 @@ aliases:
     - run:
         name: "Import signing key"
         command: |
+          gpg --keyserver keyserver.ubuntu.com \
+            --recv-keys 0x713F9F29598CFFF3 && \
           echo "${GPG_KEY}" | base64 -d > signing_key.asc && \
-          gpg --batch \
-            --passphrase "${GPG_PASSPHRASE}" \
```

Comment:
Because we're not using passphrase for this key, the `--batch` and `--passphrase` options are unnecessary.